accountpermissions/css/style.css

```diff
@@ -1,15 +1,15 @@
 body {
-	background: #000;
```

Comment:
Can you adjust the coala configuration to comply with the existing code style? We don't want to accept a huge patch for no good reason.